README.md

```diff
@@ -7,12 +7,7 @@ Standard stylelint config for Netzstrategen projects.
 Install the configuration and all peer dependencies:
 
 ```bash
-npm install --save-dev \
-  @netzstrategen/stylelint-config \
-  stylelint \
-  stylelint-order \
-  stylelint-scss \
-  stylelint-selector-bem-pattern
```

Comment:
These 3 are dependencies of this package and do not need to be installed in a project as they are in the dependency tree so i've removed them here